tests/requirements.in

```diff
@@ -0,0 +1,10 @@
+-c constraints.in # <-- contains known limitations
```

Comment:
We can remove this and then explicitly pass `constraints.in` to `pip-compile`[^1], so then we could have an open-ended requirements file (without additional constraints applied on antsibull-docs and such) to use for the `devel` branch as @samccann requested.

[^1]: the full command would be `pip-compile --allow-unsafe --output-file=requirements.txt --resolver=backtracking --strip-extras requirements.in constraints.in`

Reply:
That would be incompatible with dependabot. Also, why would we allow known incompatibilities anyway? If someone wants to, they could do a follow-up `pip install $broken-thing` command separately..

Reply:
> Also, why would we allow known incompatibilities anyway?

`antsibull-docs` is not pinned due to known incompatibilities.

Reply:
If it's not, it should just be removed from this file. But I'd shift doing so to a separate PR.

Reply:
> > Also, why would we allow known incompatibilities anyway?
> 
> `antsibull-docs` is not pinned due to known incompatibilities.

Correct me if I'm mistaken but did we not pin that here? https://github.com/ansible/ansible-documentation/pull/60/files#diff-9a2498a95a6304fab6fd43bb37c6658052a75ea6eeb9781a285e855708cf3421R5